BOJ/2212_센서/chldppwls12.py

```diff
@@ -0,0 +1,23 @@
+from sys import stdin
+input = stdin.readline
+
+n = int(input())
+k = int(input())
+num_list = list(map(int, input().split()))
+
+def solution(n, k, num_list):
+  ans = 0
+  if (k >= n):
+    return ans
+  else:
+    num_list.sort()
+    diff_list = []
+    for i in range(len(num_list)-1):
+      diff_list.append(num_list[i+1]-num_list[i])
```

Comment:
```suggestion
    diff_list = [num_list[i] - num_list[i - 1] for i in range(1,n)]
```
이런식으로 코드를 줄일 수 있습니다!